cmd/test/main_test.go

```diff
@@ -188,28 +189,28 @@ func test_testCommand_no_flags(t *testing.T, evaluator *mockEvaluator, k8sValida
 
 	k8sValidator.AssertCalled(t, "ValidateResources", mock.Anything, 100)
 	evaluator.AssertCalled(t, "CreateEvaluation", "134kh", "", "1.18.0", "Default")
-	evaluator.AssertCalled(t, "Evaluate", filesConfigurations, evaluationId)
+	evaluator.AssertCalled(t, "Evaluate", filesConfigurations, evaluationId, mock.Anything, mock.Anything, mock.Anything)
```

Comment:
This is not ideal, although I couldn't fix it myself.
If you can't fix this (as well as the other 3 occurrences, that is OK, I will try to fix it later)

Reply:
yeah, you're right. I tried my best to fix it in the ideal way. But for the lack of my knowledge in testify I couldn't fix it. 
Although, I will try to fix it now.